lib/rock/webapp/tasks/api.rb

```diff
@@ -334,6 +334,26 @@ def get_operation(name_service, name, operation_name)
                         end  
                     end
                     
+                    desc "management for running tasks ('start', 'stop', 'configure', 'cleanup')"
+                    # has to be defined after other requests e.g. ':name_service/:name/properties' or ':name_service/:name/ports'
+                    # in order to be evalueated after them, otherwise this would catch the other requests and return false  
+                    get ':name_service/:name/:action' do
+                        task = task_by_name(params[:name_service], params[:name])
+                        action = params[:action]
+                        #check for allowed actions for securiry reasons
+                        #otherwise all the tasks methods could be called using this interface
+                        if ['start', 'stop', 'configure', 'cleanup', 'reset_exception'].include? action
+                            begin
+                                task.send(action)
+                                return true
```

Comment:
The ruby function returns the last statement by default, as e.g. start has "no" return value, "nil" is returned by the rest api

my thoughts were when someone uses the function in the browser url or with curl on the command line, "nil" is displayed as result.

```
$> curl localhost:9292/tasks/tasks/localhost/planner/start
nil
```
This might lead to the assumption that the call was not successful, evne though it was.